RELEASE-NOTES.md

```diff
@@ -1,5 +1,41 @@
 # Prefect Release Notes
 
+## Release 2.13.5
+
+### Enhancements
+- Use flow run context for default values in task run logger — https://github.com/PrefectHQ/prefect/pull/10334
+- Default PREFECT_UI_API_URL to relative path /api — https://github.com/PrefectHQ/prefect/pull/10755
+- Runner webserver — https://github.com/PrefectHQ/prefect/pull/10850
+- deployment wizard: select blob storage — https://github.com/PrefectHQ/prefect/pull/10656
+- Add the ability to serve remotely stored flows — https://github.com/PrefectHQ/prefect/pull/10884
+- Vendor FastAPI as a first step toward `pydantic>2` support — https://github.com/PrefectHQ/prefect/pull/10860
+- Isolate the server-side schemas from the client-side schemas in preparation for `pydantic` v2-native constructs — https://github.com/PrefectHQ/prefect/pull/10867
+- Introduce a `pydantic` v2 compatability flag — https://github.com/PrefectHQ/prefect/pull/10868
+- Conditional imports for `pydantic` Settings — https://github.com/PrefectHQ/prefect/pull/10870
+- Conditional imports for `pydantic` throughout — https://github.com/PrefectHQ/prefect/pull/10873
+- Handle validation on `pydantic` v2 models used as parameters in tasks and flows - https://github.com/PrefectHQ/prefect/pull/10891
+- Retry on responses with a 408 status code — https://github.com/PrefectHQ/prefect/pull/10883
+
+### Fixes
+- Enforce that agents only query WorkQueues in `prefect-agent` typed WorkPools if no pool is specified — https://github.com/PrefectHQ/prefect/pull/10804
+- Preserve set attribute information on `pydantic` models used in tasks — https://github.com/PrefectHQ/prefect/pull/10876
+- Fix spaces in Python executable path — https://github.com/PrefectHQ/prefect/pull/10878
+- Update `PREFECT__FLOW_RUN_ID` environment variable to dash-delimited UUID format — https://github.com/PrefectHQ/prefect/pull/10881
+- lazily initialize semaphore for flow entrypoint discovery — https://github.com/PrefectHQ/prefect/pull/10871
+
+### Documentation
+- Add GCP Vertex AI worker to work-pools.md — https://github.com/PrefectHQ/prefect/pull/10858
+- Update pinned mkdocs-material insiders version in netlify.toml — https://github.com/PrefectHQ/prefect/pull/10859
+- Add more info about rate limits and include use cases for global concurrency — https://github.com/PrefectHQ/prefect/pull/10886
+- Change docker guide link to point to worker tutorial — https://github.com/PrefectHQ/prefect/pull/10872
+- A tale of two docker tutorials: Increasing the emphasis that workers are an optional alternative to .serve() in the tutorials — https://github.com/PrefectHQ/prefect/pull/10861
```

Comment:
These should be in user-friendly language, like: 

```
- Users can now abc the xyz
- Resolved bug that could have destroyed the world.
- Add/update X (in order) to/for/during Y
- Fix/improve to X when/where Y
```
[source](https://www.notion.so/prefect/Prefect-2-Release-Workflow-155f6a88a53c4101a9af467abf3991aa?pvs=4#fd9da6d38bc84f77916f9b9d2db97edd)

Reply:
I'm not sure how to apply that to docs changes.

Reply:
I've made some suggestions :)

Reply:
Thank you!